test/TestValidate.cpp

```diff
@@ -348,6 +334,7 @@ void TestValidate::test() {
     CPPUNIT_ASSERT_EQUAL(false, myResult.hasErrors());
     
     myResult = file.validate();
+    std::cerr << myResult << std::endl;
```

Comment:
ups, indeed some debug code. Will remove it and the rebase, when the other pr is merged...